libsplinter/Cargo.toml

```diff
@@ -66,23 +66,32 @@ glob = "0.2"
 [features]
 default = []
 
+stable = [
+    "default",
+    "events",
+    "sawtooth-signing-compat",
+]
+
 experimental = [
-    "zmq-transport",
     "biome",
+    "biome-credentials",
     "biome-notifications",
-    "database",
     "connection-manager",
+    "database",
+    "zmq-transport",
 ]
 
-connection-manager = []
-zmq-transport = ["zmq"]
-ursa-compat = ["ursa"]
-sawtooth-signing-compat = ["sawtooth-sdk"]
-events = ["hyper", "tokio", "awc"]
 biome = ["database", "jsonwebtoken", "rand"]
 biome-credentials = ["biome", "database", "bcrypt"]
 biome-notifications = ["biome", "database"]
+connection-manager = []
 database = ["diesel", "diesel_migrations"]
+events = ["hyper", "tokio", "awc"]
+sawtooth-signing-compat = ["sawtooth-sdk"]
+# The `ursa` dependency is currently broken, and will need to be fixed before
```

Comment:
Suggest moving this to its own section/list, and making it clear that it is a list of broken features.

Reply:
Updated